src/components/LocationPage/LocationPageHeader.tsx

```diff
@@ -207,7 +196,7 @@ const LocationPageHeader = (props: {
                 </SectionColumn>
               ) : (
                 <SectionColumn isUpdateCopy>
-                  <ColumnTitle isUpdateCopy>New feature</ColumnTitle>
+                  <ColumnTitle isUpdateCopy>announcement</ColumnTitle>
```

Comment:
Flag: We may want to get rid of the `usingCMSData` check above so that this new announcement is shown for all counties.